app/validators/json_validator.rb

```diff
@@ -0,0 +1,22 @@
+class JsonValidator
+  def initialize(schema, payload)
+    @schema_dir = Rails.root.join("public/schemas")
```

Comment:
Just a slight niggle about naming: I think the incoming parameter and paramter passed to `#load_schema` should be renamed from `schema` to `schema_name` to avoid any confusion

Reply:
Thanks @stephenrichards I have made the changes you requested.